pages/ireland.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="shortcut icon" href="../assets/irelandFavicon.png">
+    <title>France</title>

```

Comment:
Can you change the title to Ireland

Reply:
@harshpatel2212 just this, I can resolve the conflict for you

Reply:
ya sure